wrapanapi/systems/msazure.py

```diff
@@ -761,6 +768,24 @@ def remove_netsec_group(self, group_name, resource_group=None):
         self.logger.info("Network Security Group '%s' is removed", group_name)
         return operation.status()
 
+    def create_netsec_group_port_allow(self, group_name, port, resource_group=None):
+        secgroup_name = 'port_{:d}'.format(port)
+
+        self.logger.info("Attempting to Create New Azure Security Group '%s' "
+                         "Rule '%s'.", secgroup_name, group_name)
+
+        parameters = NetworkSecurityGroup(location=self.region)
+        parameters.security_rules = [
+            SecurityRule(
+                'Tcp', '*', '*', 'Allow', 'Inbound', description='Allow port {:d}'.format(port),
+                source_port_range='*', destination_port_range=str(port), priority=100,
+                name=secgroup_name)]
+        nsg = self.network_client.network_security_groups
+        operation = nsg.create_or_update(resource_group, group_name, parameters)
+        operation.wait()
+        self.logger.info("Network Security Group Rule is created.", group_name)
+        return operation.status()
```

Comment:
Why return this status? I would think this status should be validated before logging success, and it would be better to return the rule, or a boolean indicating success.

Reply:
Here I just followed how it is done in the other methods in this module.

Reply:
I see what you mean - we should retain this pattern for now, and if its changed it should be done as a complete change with a minor version update (ex. 3.1). Something for another PR :)